modules/install/rke2_stop_start_instace.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash  
+#set -x
+echo "$@"
+type=$1
+resource_name=$2
+
+cd /tmp
+if [ "$type" == "stop" ]; then    
```

Comment:
we should not hard code the number of nodes, instead use as var and treat that here and tf module

opinions here @rancher-max @ShylajaDevadiga  ?

Reply:
Since they are elastic ips, i would suggest let's limit the count to 3 server and 1 agent.